### PR TITLE
DM-47000: Turn on verification tests and fix verification test code

### PR DESCRIPTION
**Pull request by erykoff**
The verification test code was not handling sub-dicts and sub-lists correctly, and that has now been fixed.  The test values have also been updated.

Note: the bias noise tests are now being marked as success: false by cp_verify, I'm assuming due to the units fix.

### CODE REVIEW

File: tests/data/flatRun.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 2021052500077:
-  FAILURES:
-  - RXX_S00 C07 NOISE
+  SUCCESS: false
```
Comment (czwa) [inline tests/data/flatRun.yaml:2]: I'm not sure I understand this.  `SUCCESS: false` should be followed by `FAILURES: [not-an-empty-list]`.  I'm sure this is what the source file had, and I'll have a look once this ticket merges to see if I can understand this more.

File: tests/data/biasRun.yaml [hunk 2]
```diff
@@ -1,16 +1,21 @@
 2021052500015:
+  SUCCESS: false
```
Comment (czwa) [inline tests/data/biasRun.yaml:2]: The values in the results dictionary show this is getting a value of ~125, so there's a bug somewhere.  DM-47023 is the ticket to track down this change.

**Review by czwa — APPROVED** [review]
I don't fully understand the emergent bug in the bias, but I don't think it's a critical failure (as turning on clipped stats immediately tosses out whatever bright pixels are contaminating the calculation).

File: tests/data/flatRun.yaml (hunk 1 above)
Comment (erykoff) [reply tests/data/flatRun.yaml:2]: Oh huh, good point.  I'm just copying in what it has but I agree it doesn't make sense.

**Pull request merged**